plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/prior_box.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/prior_box.hpp"
+
+using LayerTestsDefinitions::PriorBoxLayerTest;
+
+namespace {
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    // InferenceEngine::Precision::FP16,
+};
+
+// The count of tests will be too large for CI, close most checks here
+const std::vector<std::vector<float>> minSizes = {{256.0f}};
+const std::vector<std::vector<float>> maxSizes = {{315.0f}};
+const std::vector<std::vector<float>> aspectRatios = {
+    {1.0f},
+    // {2.0f, 3.0f}
```

Comment:
Just to reduce the amount of tests, there will be 700+ tests for this op and in fact 1400+ if I add more details,  these may take hours on my side, so not sure if it worth to open all tests as i locally passed all of them.

Reply:
Open all tests and just keep a few in smoke tests

Reply:
I'm a little worried about how slow these tests are running. I would not expect PriorBox to be especially fast, but it's slower than I hoped for. I wonder if perhaps there's a way to use additional dimensions when constructing the output (perhaps a dimension for x vs y, a dimension for which aspect ratio we're on, etc) that would enable us to avoid some of the splits/concatenations currently required...

Reply:
I divide the tests to three parts which with some fixed arguments to reduce size,  it can cover most classic combinations and now only 76 tests, it takes 2 minutes on my side. As i have confirmed the whole combinations which covers all paths can be passed, i think it is enough now.